src/components/profile/ProfileDetails.vue

```diff
@@ -260,5 +297,56 @@ export default defineComponent({
         />
       </div>
     </div>
+
+    <!-- Title -->
+    <h2
+      class="text-h6 text-grey-10 q-mb-none q-mt-xl"
+      data-cy="profile-title-registration-details"
+    >
+      {{ $t('profile.titleRegistrationDetails') }}
+    </h2>
+    <!-- Registration details -->
+    <div class="q-mt-lg">
+      <div class="row q-col-gutter-lg">
+        <!-- Package -->
+        <details-item
+          :label="$t('profile.labelPaymentState')"
+          :value="profile.package.title"
+          class="col-12 col-md-6 items-center"
+          data-cy="profile-details-payment-state"
+        >
+          <template #value>
+            <div class="row q-col-gutter-md justify-between">
+              <div class="col-12 col-sm-auto flex items-center gap-8">
+                <q-icon
+                  :name="iconPaymentState"
+                  size="18px"
```

Comment:
`18px` repeat, you can use `const`, please.

Reply:
Updated using const.